webapp/channels/src/components/suggestion/at_mention_provider/at_mention_suggestion.tsx

```diff
@@ -175,8 +187,10 @@ const AtMentionSuggestion = React.forwardRef<HTMLLIElement, SuggestionProps<Item
     return (
         <SuggestionContainer
             ref={ref}
+            role='option'
             {...props}
             data-testid={`mentionSuggestion_${itemname}`}
+            id={`post_textbox_@${itemname}`}
```

Comment:
This is only going to work in the post textbox in the centre channel, not in the RHS (or other places where this autocomplete is used). I think we're going to need to pass the ID of the textbox from the `SuggestionBox` through to the `SuggestionList` and then through to this component